poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFSheet.java

```diff
@@ -100,11 +101,6 @@ public InputStream getWorksheetXMLInputStream() throws IOException {
     }
 
     //start of interface implementation
-    @Override
-    public Iterator<Row> iterator() {
-        return rowIterator();
```

Comment:
ditto - please add this back

Reply:
I've moved it to the `Sheet` parent interface, so it essentially the same (alias for `rowIterator()`), see https://github.com/apache/poi/pull/290/files#diff-c77e303b24d9da0a72d46cf8ae7319ba6876b1b5c7d89e6ddc51acea6a6571ce